apple/internal/processor.bzl

```diff
@@ -452,6 +454,7 @@ def _bundle_partial_outputs_files(
             input_manifests = resolved_bundletool.input_manifests,
             mnemonic = "BundleApp",
             progress_message = "Bundling %s" % label_name,
+            env = env,
```

Comment:
[question] Why is `env` needed here? I do not see the IPA post-processor used around here

Reply:
I added env to similar code in `codesigning_support`. Then I noticed, that inside`_bundle_partial_outputs_files` method there is `post_processor` that was added to `bundling_tools` above (line 416). Also `post_processor_path` is a parameter for `control` structure, that is written to `actions` (lines 378 and 395). And `_bundle_partial_outputs_files` method is called by `_bundle_post_process_and_sign`. So I decided to add the code here for consistency.

Reply:
Not sure this should be added in this PR. If you add `env` to some places that are unrelated with `IPA post-processor` is great, but you would also need to add tests for the changes. 

I think there are two options:
* Add only `env` where needed by the PR (with tests)
* Add `env` everywhere where it is missing, adding tests for all cases